src/components/options/SpcOptionEditor.tsx

```diff
@@ -17,11 +17,19 @@ const aggregationTypeOptions: Array<{ label: string; value: AggregationType }> =
   { label: 'Range', value: 'range' },
   { label: 'Standard deviation', value: 'standardDeviation' },
 ];
+const chartTypeOptions: Array<{ label: string; value: ChartType }> = [
+  { label: 'Mean chart', value: 'meanChart' }, //default
+  { label: 'Moving-range chart', value: 'mrChart' },
```

Comment:
label: 'Moving-range chart' ->  label: 'Moving-range'

Reply:
done in [cf90bec](https://github.com/KensoBI/spc-panel/pull/48/commits/cf90bec0af252cb0846e58b2ba2280674d256aaf)